policyengine_canada/variables/household/income/household/household_benefits.py

```diff
@@ -0,0 +1,17 @@
+from policyengine_canada.model_api import *
+
+
+class household_benefits(Variable):
+    value_type = float
+    entity = Household
+    label = "benefits"
+    unit = CAD
+    definition_period = YEAR
+    adds = [
+        "Canada_child_benefit",
```

Comment:
is that what the variables are called in the repo?

Reply:
the file is named "ccb"

Reply:
> is that what the variables are called in the repo?

the file is named "ccb"

Reply:
I think the variable is just called 
`child_benefit`
`dental_benefit`